data/mods/Magiclysm/traits/attunements.json

```diff
@@ -0,0 +1,1183 @@
+[
+  {
+    "id": "ARTIFICER",
+    "type": "mutation",
+    "name": { "str": "Artificer" },
+    "points": 8,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "description": "With the mechanically-minded Technomancer and the sheer power of lightning from a Stormshaper, the Artificer has a unique confluence of skills and spells to be perfect for invention.  While modern inventions and machines have won out in a modern society, after a cataclysm one has a chance to go back to roots of things.  To make weird gadgets that are just as likely to explode as explode the thing you're pointing at.",
+    "prereqs": [ "STORMSHAPER", "TECHNOMANCER" ],
+    "cancels": [
+      "AURA_MAGE",
+      "BIOTEK",
+      "BOREAL_MAGE",
+      "CRUSADER",
+      "CLEANSING_FLAME",
+      "EARTH_ELEMENTAL",
+      "FIRE_ELEMENTAL",
+      "FORCE_MAGE",
+      "GAIAS_CHOSEN",
+      "GLACIER_MAGE",
+      "GOLEMANCER",
+      "GRAVITY_MAGE",
+      "OVERCLOCKER",
+      "ICE_ELEMENTAL",
+      "ILLUSIONIST",
+      "MAGNETISM_MAGE",
+      "PERMAFROST_MAGE",
+      "RADIATION_MAGE",
+      "STORM_ELEMENTAL",
+      "STORMCALLER",
+      "SUN_MAGE",
+      "TUNDRA_MAGE",
+      "VOID_MAGE",
+      "VULCANIST",
+      "SOULFIRE"
+    ],
+    "flags": [ "ATTUNEMENT" ]
+  },
+  {
+    "id": "AURA_MAGE",
+    "type": "mutation",
+    "name": { "str": "Aura Mage" },
+    "points": 8,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "description": "",
```

Comment:
Will it be added later?

Reply:
yes it will. i've got a whole bunch of commits on a separate branch and realized there was too much C++ to stick it in one big JSON PR. at the very latest it'll be fixed before the attunements "go live". that is, players able to get them in the game.